src/main/java/edu/gatech/cs2340/trydent/OneTimeEvent.java

```diff
@@ -0,0 +1,75 @@
+package edu.gatech.cs2340.trydent;
+
+
+/**
+ * Creates an event that starts, updates, and stops exactly once.
+ *
+ */
+public abstract class OneTimeEvent {
```

Comment:
Okay, I'll replace all usage of this class with runLater.
I apparently misread the runLater code as being one off for inexplicable some reason.